client/src/components/Home/AboutUs/AboutUs.css

```diff
@@ -0,0 +1,104 @@
+.heading{
```

Comment:
You may add `.about-us-section` for every class or tags you select in this file to avoid clashes between similar classnames in other components.

Reply:
Yaa sure and thanks for pointing this out.